examples/channels-workers/channels_workers.description

```diff
@@ -0,0 +1,2 @@
+//Channels can be used for inter worker communications and synchronising workers.
```

Comment:
Shall we change as follows?
"//You can use channels for inter-worker communication and also to synchronize workers."